### ISSUE TITLE
Feature request: Convert DDS to image

### ISSUE THREAD

**Issue by mylastsong18** [Tichau/FileConverter#113]
As a developer I work with large amounts of DDS images and am constantly having to use Photoshop just to convert DDS to other formats such as PNG to TGA. Can you please add DDS as an input file?

**Comment by Pinkyo2386** [comment]
Yeah same issue, tried this out for DDS, still doesn't support from what i can tell
